### DISCUSSION TITLE
Share header results

### DISCUSSION THREAD

**Discussion by TheSteemer** [microsoft/MHA#640] [Ideas]
Feature request
Have the ability to share the page thats generated after the analyze headers. So it can be sent to customers or other IT folks for further diag on their ends

**Comment by stephenegriffin** [comment]
To do that would require us to persist the data on the server. As it stands right now, that data is only held client side, with nothing sent to the server.

**Comment by TheSteemer** [comment]
Maybe some kind of export options then?
